app/apollo/resolvers/channel.js

```diff
@@ -577,21 +600,16 @@ const channelResolvers = {
           }
         }
 
+        if (!channel) {
+          throw new NotFoundError(context.req.t('Channel uuid "{{channel_uuid}}" not found.', {'channel_uuid':channel_uuid}), context);
+        }
+
```

Comment:
The retrieval of the existing channel, auth check, and check for existence were moved 'up' in the function, before checking if `EXPERIMENTAL_GITOPS_ALT` would block further execution, and before verifying the `org` even exists.  This shouldn't be done -- the auth/existence checks should be kept in the same place.

Reply:
Reasoning being our conversation about checking auth before throwing validation errors that could give more information in those errors messages than should be allowed before auth is validated.

Reply:
Unresolving, not addressed.  Moving the `// get org` check to after tha auth check might be reasonable, but the expensive queries/checks should be after the simple checks like `EXPERIMENTAL_GIT_OPS`

Reply:
That would be because this is outdated

Reply:
I think the fault was on my end -- refreshing a diff between specific commits rather than getting the overall diff for the PR fresh.
Resolved.